pytorch_lightning/loggers/base.py

```diff
@@ -173,6 +174,20 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None):
         """
         pass
 
+    def log_figure(self, name: str, figure: plt.figure, step: Optional[int] = None, close: bool = True) -> None:
```

Comment:
I think we should add additional `**kwargs` here.
When the user knows they are using a particular logger that supports additinal arguments, we should allow it to be passed down.

for example, in one of the loggers you have a description, and this could be customized then by the user.
This would then still be logger agnostic, where loggers simply ignore unknown `**kwargs` when it's not applicable.

Reply:
Funnily I had it including **kwargs two times, and changed it back and forth ...

I do not like it because:
- Silently ignoring arguments that are unexpected is not what python does by default when you parse **kwargs
- It'll most certainly break for `LoggerCollection` unless one implements a more complex logic which then makes the whole implementation unnecessary

Reply:
I also think `**kwargs` is really necessary here. Each logger can have many different arguments

> Silently ignoring arguments that are unexpected is not what python does by default when you parse **kwargs

I don't get this

Reply:
If you add **kwargs it will break LoggerCollection if you don't do anything. So what do you do if you have logger A that can take a certain argument that logger B in your collection does not? Ignore argument, raise?
Or add more complex logic that looks something like this

```python
my_mwargs = {
    'loggerA': {'kwarga': 1},
    'loggerB': {'kwargb': 2},
}
```

Reply:
--> Needs design decision